src/herder/TxSetFrame.h

```diff
@@ -119,7 +119,10 @@ class TxSetFrame : public AbstractTxSetFrameForApply
     void toXDR(TransactionSet& set) override;
 };
 
-// 3-way fee rate compare
+// 3-way fee rate compare variants
+int feeRate3WayCompare(int64 lFeeBid, uint32 lNbOps, int64 rFeeBid,
```

Comment:
Nitpick: these functions are defined in a strange place given that they don't really have anything to do with transaction set frames. I spent a while looking for them because I couldn't remember the name and they weren't where I would have guessed. Maybe put them in TransactionFrameBase.h or something?

Reply:
sure done